docs/manuals/source/Appendix/Debugging.rst

```diff
@@ -168,16 +168,38 @@ Our Linux packages are stripped of debugging symbols, so you need an extra step
 
 
    .. code-block:: shell-session
-      :caption: Installing Bareos debug symbols on (RH)EL system
+      :caption: Installing Bareos Director debug symbols on (RH)EL system
 
-      dnf --enablerepo bareos-debuginfo install bareos-director-debuginfo
+      dnf --enablerepo=bareos-debuginfo install gdb bareos-debuginfo bareos-director-debuginfo \
+      bareos-database-common-debuginfo bareos-common-debuginfo bareos-database-tools-debuginfo
```

Comment:
This seems to mostly work except that somehow it cannot find the files themselves.  For debug traces this is good enough, but we should still look into why the files are missing.  E.g.
```
(gdb) bt
#0  0x00007f17e90797fd in ReadStateFile (dir=<optimized out>, progname=<optimized out>, port=<optimized out>) at ../../../../core/src/lib/bsys.cc:573
#1  0x000055d3d46385e0 in main (argc=2, argv=0x7ffdf3cf2468) at ../../../../core/src/dird/dird.cc:306
(gdb) info locals
filename = "/var/lib/bareos/bareos-dir.9101.state"
secure_erase_guard = {filename = "/var/lib/bareos/bareos-dir.9101.state", cleanup = true}
hdr = {id = '\000' <repeats 13 times>, version = 0, last_jobs_addr = 0, end_of_recent_job_results_list = 0, reserved = {0 <repeats 19 times>}}
file = {<std::basic_istream<char, std::char_traits<char> >> = {<std::basic_ios<char, std::char_traits<char> >> = {<std::ios_base> = {_vptr.ios_base = 0x7f17e8b8b998 <vtable for std::basic_ifstream<char, std::char_traits<char> >+64>, static boolalpha = std::_S_boolalpha, static dec = std::_S_dec, static fixed = std::_S_fixed, static hex = std::_S_hex, static internal = std::_S_internal, static left = std::_S_left, 
        static oct = std::_S_oct, static right = std::_S_right, static scientific = std::_S_scientific, static showbase = std::_S_showbase, static showpoint = std::_S_showpoint, static showpos = std::_S_showpos, static skipws = std::_S_skipws, static unitbuf = std::_S_unitbuf, static uppercase = std::_S_uppercase, static adjustfield = std::_S_adjustfield, static basefield = std::_S_basefield, 
        static floatfield = std::_S_floatfield, static badbit = std::_S_badbit, static eofbit = std::_S_eofbit, static failbit = std::_S_failbit, static goodbit = std::_S_goodbit, static app = std::_S_app, static ate = std::_S_ate, static binary = std::_S_bin, static in = std::_S_in, static out = std::_S_out, static trunc = std::_S_trunc, static beg = std::_S_beg, static cur = std::_S_cur, static end = std::_S_end, 
        _M_precision = 6, _M_width = 0, _M_flags = 4098, _M_exception = 5, _M_streambuf_state = std::_S_failbit, _M_callbacks = 0x0, _M_word_zero = {_M_pword = 0x0, _M_iword = 0}, _M_local_word = {{_M_pword = 0x0, _M_iword = 0}, {_M_pword = 0x0, _M_iword = 0}, {_M_pword = 0x0, _M_iword = 0}, {_M_pword = 0x0, _M_iword = 0}, {_M_pword = 0x0, _M_iword = 0}, {_M_pword = 0x0, _M_iword = 0}, {_M_pword = 0x0, _M_iword = 0}, {
            _M_pword = 0x0, _M_iword = 0}}, _M_word_size = 8, _M_word = 0x7ffdf3cf1a70, _M_ios_locale = {static none = 0, static ctype = 1, static numeric = 2, static collate = 4, static time = 8, static monetary = 16, static messages = 32, static all = 63, _M_impl = 0x7f17e8b93c80 <(anonymous namespace)::c_locale_impl>, static _S_classic = 0x7f17e8b93c80 <(anonymous namespace)::c_locale_impl>, 
          static _S_global = 0x7f17e8b93c80 <(anonymous namespace)::c_locale_impl>, static _S_categories = 0x7f17e8b86460 <__gnu_cxx::category_names>, static _S_once = 2, static _S_twinned_facets = 0x7f17e8b87c00 <std::locale::_S_twinned_facets>}}, _M_tie = 0x0, _M_fill = 0 '\000', _M_fill_init = false, l_M_streambuf = 0x7ffdf3cf1940, _M_ctype = 0x7f17e8b936a0 <(anonymous namespace)::ctype_c>, 
      _M_num_put = 0x7f17e8b93630 <(anonymous namespace)::num_put_c>, _M_num_get = 0x7f17e8b93640 <(anonymous namespace)::num_get_c>}, _vptr.basic_istream = 0x7f17e8b8b970 <vtable for std::basic_ifstream<char, std::char_traits<char> >+24>, _M_gcount = 0}, _M_filebuf = {<std::basic_streambuf<char, std::char_traits<char> >> = {
      _vptr.basic_streambuf = 0x7f17e8b8b878 <vtable for std::basic_filebuf<char, std::char_traits<char> >+16>, _M_in_beg = 0x0, _M_in_cur = 0x0, _M_in_end = 0x0, _M_out_beg = 0x0, _M_out_cur = 0x0, _M_out_end = 0x0, _M_buf_locale = {static none = 0, static ctype = 1, static numeric = 2, static collate = 4, static time = 8, static monetary = 16, static messages = 32, static all = 63, 
        _M_impl = 0x7f17e8b93c80 <(anonymous namespace)::c_locale_impl>, static _S_classic = 0x7f17e8b93c80 <(anonymous namespace)::c_locale_impl>, static _S_global = 0x7f17e8b93c80 <(anonymous namespace)::c_locale_impl>, static _S_categories = 0x7f17e8b86460 <__gnu_cxx::category_names>, static _S_once = 2, static _S_twinned_facets = 0x7f17e8b87c00 <std::locale::_S_twinned_facets>}}, _M_lock = {__data = {__lock = 0, 
        __count = 0, __owner = 0, __nusers = 0, __kind = 0, __spins = 0, __elision = 0, __list = {__prev = 0x0, __next = 0x0}}, __size = '\000' <repeats 39 times>, __align = 0}, _M_file = {_M_cfile = 0x0, _M_cfile_created = false}, _M_mode = 0, _M_state_beg = {__count = 0, __value = {__wch = 0, __wchb = "\000\000\000"}}, _M_state_cur = {__count = 0, __value = {__wch = 0, __wchb = "\000\000\000"}}, _M_state_last = {
      __count = 0, __value = {__wch = 0, __wchb = "\000\000\000"}}, _M_buf = 0x0, _M_buf_size = 8192, _M_buf_allocated = false, _M_reading = false, _M_writing = false, _M_pback = 0 '\000', _M_pback_cur_save = 0x0, _M_pback_end_save = 0x0, _M_pback_init = false, _M_codecvt = 0x7f17e8b93610 <(anonymous namespace)::codecvt_c>, _M_ext_buf = 0x0, _M_ext_buf_size = 0, _M_ext_next = 0x0, _M_ext_end = 0x0}}
(gdb) list
568	in ../../../../core/src/lib/bsys.cc
```